tools/c7n_mailer/c7n_mailer/cli.py

```diff
@@ -39,6 +39,7 @@
         'queue_url': {'type': 'string'},
         'endpoint_url': {'type': 'string'},
         'from_address': {'type': 'string'},
+        'additional_email_headers': {'type': 'object'},
```

Comment:
probably want a pattern props here to avoid this being a full wild card, ie expect values to be strings.

Reply:
Ok, thanks. I hadn't even known about support for ``patternProperties``. Updated commit coming.

Reply:
This has been fixed in the latest commit.